packages/core/contracts/polygon/OracleBaseTunnel.sol

```diff
@@ -16,14 +15,25 @@ abstract contract OracleBaseTunnel {
         int256 price;
     }
 
+    // This value must be <= the Voting contract's `ancillaryBytesLimit` value otherwise it is possible
+    // that a price can be requested to this contract successfully, but cannot be resolved by the DVM which refuses
+    // to accept a price request made with ancillary data length over a certain size.
+    uint256 public constant ancillaryBytesLimit = 8192;
+
     // Mapping of encoded price requests {identifier, time, ancillaryData} to Price objects.
     mapping(bytes32 => Price) internal prices;
 
     // Finder to provide addresses for DVM system contracts.
     FinderInterface public finder;
 
-    event PriceRequestAdded(bytes32 indexed identifier, uint256 time, bytes ancillaryData, bytes32 requestHash);
-    event PushedPrice(bytes32 indexed identifier, uint256 time, bytes ancillaryData, int256 price, bytes32 requestHash);
+    event PriceRequestAdded(bytes32 indexed identifier, uint256 time, bytes ancillaryData, bytes32 indexed requestHash);
```

Comment:
- The [`OracleBaseTunnel` events](https://github.com/UMAprotocol/protocol/blob/d146d4feddcf99e157a909032a3071e95ca51eb9/packages/core/contracts/polygon/OracleBaseTunnel.sol#L25-L26) do not index the `requestHash` parameter, even though this parameter would naturally be used to track the request.